### PR TITLE
Added usage manual for those using Redis ACL

### PR DESCRIPTION
**Pull request by raflymln**
When i try to connect to Redis using connection string, it's not allowing me to connect to another user than `default` user, so i spent 2 days to find out the problem just to found that my Redis config is fine, and i just need to connect using `RATE_LIMITER_REDIS_*` and `CACHE_REDIS_*` variables.

Even though it SHOULD allow us to connect to Redis, that using ACL, with connection string, as stated on https://github.com/luin/ioredis#connect-to-redis, however the only way to allow us to connect to different user than `default` user is with object only.

### CODE REVIEW

**Comment by erondpowell** [comment]
@br41nslug I haven't messed around with Redis config stuff at all yet. Can you (and do you have time) to review this?

**Review by br41nslug — COMMENTED** [review]
After some research i found that this issue in `ioredis` was fixed here in the v5 update so rather than updating the docs to work around this issue I've created a PR to update `ioredis` so the connection URIs work as expected 😄 
ioredis issue: https://github.com/luin/ioredis/commit/c6f41f692243129dbc952ef8fd2e5c160133d677
https://github.com/directus/docs/pull/153
directus PR: https://github.com/directus/directus/pull/15876

File: self-hosted/config-options.md [hunk 1]
```diff
@@ -408,7 +416,15 @@ RATE_LIMITER_DURATION="5"
 
 RATE_LIMITER_STORE="redis"
 
+# If you are connecting to Redis `default` user
```
Comment (br41nslug) [inline self-hosted/config-options.md:419]:
```suggestion
```

File: self-hosted/config-options.md [hunk 2]
```diff
@@ -472,17 +488,25 @@ No additional configuration required.
 
 ### Redis
 
+::: danger Using Redis ACL
+
+If you are using Redis Access Control List (ACL) system, you need to use `CACHE_REDIS_*` variables instead of `CACHE_REDIS`. Using Redis connection string will only allow you to authenticate to Redis `default` user.
+
+:::
+
```
Comment (br41nslug) [inline self-hosted/config-options.md:496]:
```suggestion
```

**Review by br41nslug — APPROVED** [review]
LGTM :) can be merged when https://github.com/directus/directus/pull/15876 is merged.

**Comment by raflymln** [comment]
> LGTM :) can be merged when [directus/directus#15876](https://github.com/directus/directus/pull/15876) is merged.

Thank you!

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**